src/theme-context-provider.tsx

```diff
@@ -90,13 +90,12 @@ const ThemeContextProvider: React.FC<Props> = ({theme, children, as}) => {
     const isDarkModeEnabled = (colorScheme === 'auto' && isOsDarkModeEnabled) || colorScheme === 'dark';
     const colors: Colors = isDarkModeEnabled ? darkColors : lightColors;
 
-    const contextTheme = React.useMemo<Theme>(() => {
+    const contextTheme = React.useMemo((): Theme => {
```

Comment:
This way the function return type is stricter. Otherwise it accepts an object with unexpected properties.